src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1439,7 +1439,9 @@ Base.hcat(df1::AbstractDataFrame, df2::AbstractDataFrame, dfn::AbstractDataFrame
 """
     vcat(dfs::AbstractDataFrame...;
          cols::Union{Symbol, AbstractVector{Symbol},
-                     AbstractVector{<:AbstractString}}=:setequal)
+                     AbstractVector{<:AbstractString}}=:setequal,
+         indicator::Union{Nothing, SymbolOrString,
+                          Pair{<:SymbolOrString, <:AbstractVector}}=nothing)
```

Comment:
We don't export `SymbolOrString`:
```suggestion
         indicator::Union{Nothing, Symbol, AbstractString,
                          Pair{<:Union{Symbol, AbstractString}, <:AbstractVector}}=nothing)
```